packages/bpk-component-button/src/BpkButton-test.js

```diff
@@ -17,10 +17,25 @@
  */
 
 import React from 'react';
+import { axe } from 'jest-axe';
 import renderer from 'react-test-renderer';
+import { render } from '@testing-library/react';
```

Comment:
> To make it work, I had to add a dependency on `@testing-library/react`, as `jest-axe` won't work with react-test-renderer. This is because it requires real HTML, not the pseudo-HTML structures generated by our tests.

While, for the reasons stated in the PR description, I agree about using a) only one test framework and b) the ideal would be `@testing-library/react` we don't _have_ to introduce it in this PR.

It looks like the Backpack codebase has both `react-test-renderer` and `enzyme` already installed.

`enzyme` can be used as so: https://github.com/nickcolley/jest-axe#testing-react-with-enzyme

while it's also possible to do `ReactDOMServer.renderToString`: https://codeburst.io/automated-accessibility-testing-tool-a11y-pa11y-jest-storybook-2ad294bfe71a

If we're committed to the migration to `@testing-library/react` then there is no time like the present to start, but if we're unsure then `jest-axe` is not blocked on it AFAIK.

Reply:
Oh thanks, that's good to know!